src/jailer/src/env.rs

```diff
@@ -210,14 +210,11 @@ impl Env {
             .map_err(|e| Error::ChangeFileOwner(path_buf, e))
     }
 
-    pub fn run(mut self) -> Result<()> {
+    fn prepare_for_chroot(&mut self) -> Result<()> {
```

Comment:
The name is a bit misleading, this method doesn't do all the necessary preparing for chroot (cgroups, open /dev/null). Could the 2 things that this method deals with be split in different methods and tested separately?